openapi/webhooks_spec.yaml

```diff
@@ -344,85 +344,6 @@ components:
         path: description
         op: replace
         value: new description
-    Result:
-      type: object
```

Comment:
Can the specs be updated to reflect what the health check endpoint now returns for 200 & 503?

Reply:
it hasn't changed as far as I understand. we just actually returning 503 if it's unhealthy

Reply:
👍